data/Psi4/basicPsi4-1.2.1/water_mp2.in

```diff
@@ -7,6 +7,14 @@ H 1 0.99 2 106.0
 
 set {
     basis sto-3g
+    print_basis true
+    print_mos true
+    cubeprop_tasks ['orbitals', 'density']
+    cubic_grid_spacing [0.5, 0.5, 0.5]
+    cubic_grid_overage [3, 1.5, 2.4]
 }
 
-energy('mp2')
+mp2_e, mp2_wfn = energy('mp2', return_wfn=True)
```

Comment:
Change this to
```suggestion
mp2_e, scf_wfn = energy('mp2', return_wfn=True)
```
because this doesn't actually calculate the MP2 wavefunction, only the Hartree-Fock wavefunction.

Reply:
I see. Thanks for letting me know. I'll change this to scf_wfn.

Reply:
I've made the necessary changes here. Thanks!